src/settings/slider-setting/slider-setting.ts

```diff
@@ -0,0 +1,70 @@
+import {UIPSetting} from '../setting/setting';
+import {attr} from '@exadel/esl/modules/esl-base-element/core';
+import {WARNING_MSG} from '../../utils/warning-msg';
+
+export class UIPSliderSetting extends UIPSetting {
+  public static is = 'uip-slider-setting';
+  /** CSS Class added when setting has inconsistent state. */
+  public static inconsistencyClass = 'inconsistency-marker';
+
+  /** Setting's visible name. */
+  @attr({defaultValue: ''}) public label: string;
+  /** Minimum range value. */
+  @attr({defaultValue: '0'}) public min: string;
+  /** Maximum range value. */
+  @attr({defaultValue: '100'}) public max: string;
+  /** Step for range. */
+  @attr({defaultValue: '1'}) public step: string;
+  /** Range input to change setting's value. */
+  protected $field: HTMLInputElement;
+  /** Range value displayed below. */
+  protected $fieldValue: HTMLDivElement;
+
+  protected connectedCallback() {
+    super.connectedCallback();
+    if (this.$field) return;
```

Comment:
I see a little bit of inconsistency here, we assume that the this.$field to be empty but at the same time
- we clear everything inside setting on the 30th line
- and we, basically, have no disconnected callback to make the field empty...

So as I see right now this custom element can not properly control its lifecycle.